UI/src/components/widgets/build/directives/buildsPerDay.html

```diff
@@ -2,7 +2,7 @@
     <div class="widget-heading">
         Builds per day
     </div>
-    <div hide-empty="['{{ buildView.lineData }}']">
+    <div hide-empty="['{{ buildView.lineData.series[0].data }}', '{{ buildView.lineData.series[1].data }}']">
```

Comment:
@kheinz04 check for undefined or null or size

Reply:
undefined assigned automatically through angular